### PR TITLE
OSD: Add reprojection camera tilt to OSD menu

### LINKED ISSUE #2033: Set Reprojection Camera Tilt on OSD
* Seen in version (release version or githash): 
* On flight controller (if applicable): 

Issue details:
Please add the ability to set Reprojection Camera Tilt to the OSD menu.

### PR DESCRIPTION
**Pull request by tracernz**
Fixed a couple of small bugs along the way.

Fixes #2033

### CODE REVIEW

**Comment by tracernz** [comment]
img_1195

**Comment by tracernz** [comment]
build artifacts please

File: flight/Modules/OnScreenDisplay/osd_menu.c [hunk 1]
```diff
@@ -1650,7 +1658,7 @@ void draw_hscale(int x1, int x2, int y, float val_min, float val_max, float val)
 	int width = x2 - x1;
 	int width2;
 	write_filled_rectangle_lm(x1, y, width, 6, 0, 1);
-	width2 = LIMIT((float)(width - 2) * val / (val_max - val_min), 0, width - 2);
+	width2 = LIMIT((float)(width - 2) * (val - val_min) / (val_max - val_min), 0, width - 2);
```
Comment (mlyle) [inline flight/Modules/OnScreenDisplay/osd_menu.c:1661]: nice catch.

**Comment by mlyle** [comment]
awesome :D

**Comment by droney** [comment]
curious... why would you specify a negative camera angle? In GCS it's always been positive and presumably between 0 and 90 degrees.

**Comment by tracernz** [comment]
GCS limits are [-85,+85]

**Comment by mlyle** [comment]
Negative "camera" reproject is useful for VTOL

**Pull request merged**